CMakeLists.txt

```diff
@@ -9,16 +9,36 @@
 
 project(KDABBoatDemo)
 cmake_minimum_required(VERSION 3.9)
-find_package(Qt5 CONFIG REQUIRED COMPONENTS Core Gui Quick)
+
+find_package(Qt6 6.3.0 COMPONENTS Core Quick Core5Compat REQUIRED)
+
+set(QT_WASM_INITIAL_MEMORY "64MB")
+
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_AUTOMOC TRUE)
 set(CMAKE_AUTORCC TRUE)
 
-add_executable(KDABBoatDemo
+set(PROJECT_SOURCES
```

Comment:
Is the intermediate variable needed? You could just pass the list of sources to `add_executable()` as before.